fem/eltrans.hpp

```diff
@@ -277,6 +278,7 @@ class InverseElementTransformation
         ip0(NULL),
         init_guess_type(Center),
         qpts_type(Quadrature1D::OpenHalfUniform),
+        refiner(qpts_type),
```

Comment:
Possible cleanup: qpts_type has no use anymore, so we can remove it from the class.